test/hotspot/jtreg/compiler/codecache/stress/OverloadCompileQueueTest.java

```diff
@@ -39,6 +39,7 @@
  *                   -XX:CompileCommand=exclude,java.lang.Thread::sleepNanos
  *                   -XX:CompileCommand=exclude,java.lang.Thread::beforeSleep
  *                   -XX:CompileCommand=exclude,java.lang.Thread::afterSleep
+ *                   -XX:CompileCommand=exclude,java.util.concurrent.TimeUnit::toNanos
```

Comment:
I'm guessing these changes have something to do with JDK-8279653?

Reply:
It should have been added when Thread.sleep was changed but we got lucky.